events/janus_wsevh.c

```diff
@@ -372,33 +380,21 @@ int janus_wsevh_init(const char *config_path) {
 		protocols[0].name = g_strdup(item->value);
 
 	/* Connect */
-	JANUS_LOG(LOG_VERB, "WebSocketsEventHandler: Connecting to WebSockets server...\n");
 	gboolean secure = !strcasecmp(protocol, "wss");
 	struct lws_context_creation_info info = { 0 };
 	info.port = CONTEXT_PORT_NO_LISTEN;
 	info.protocols = protocols;
 	info.gid = -1;
 	info.uid = -1;
+	info.timeout_secs = 10;
```

Comment:
have you also checked out `connect_timeout_secs` ?

Reply:
is only available in >= 3.2 so i just left this one here as it is available in all versions of lws.
But as mentioned before here or in the other ticket i played with so many timing options in such a wide manner but did not get the lws_service to wake up... It´s the same behaviour actually also in the earlier versions so i don´t know where the reconnect in the service loop comes from (i did not blame it :)). But i guess that the eventing websocket is not widley used as the doubling of the time on connect fails each time is such a blocker to use it productive. I would not fix it for wsl earlier than 3.2, i just ensure that it compiles but the behaviour for the old one is the same. Wakeup every 25 seconds.

Reply:
I guess that eventing websocket is not widely used _with an unreachable host_.